_articles/sc-acceptable-use.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Synapse Scientific Computing Acceptable Use
+layout: article
+excerpt: Computing resources through Synapse requires agreement with appropriate use
+category: scientific-computing
```

Comment:
This is not a category in the docs - therefore this page will not get surfaced on the website when the PR is approved. Is that behavior desired? The possible categories are the sections on [this page](https://docs.synapse.org/articles/article_index.html) and the syntax for the yaml header is determined from the list [here](https://github.com/Sage-Bionetworks/synapseDocs/tree/gh-pages/_categories).

Reply:
I see the other PR for this category.